client/tailwind.config.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
       ...theme("colors"),
       primary: "#00204E",
       secondary: "#FFFFFF",
-      hover: "#00B5B5",
+      hover: "#0060de",
+      selected: "#00B5B5",
```

Comment:
Minor: I would keep the `hover` as what it was before (#00B5B5) and change the `selected` to (#0171b7).

It'll be something like this in the end which keeps it a bit more consistent with the colors scheme we're using.

![image](https://user-images.githubusercontent.com/24628243/194735784-00fd364e-bcbb-4bb2-b6f2-6c7b09af78c4.png) - Top is `selected` and bottom is `hover`

Reply:
Thanks. Color choice is not my strong suit. :laughing: